gov2/aurora/README.md

```diff
@@ -118,9 +121,9 @@ in the `gov2` folder.
 
 ## Additional resources
 
-* [Aurora User Guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_AuroraOverview.html)
-* [Aurora API Reference](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/Welcome.html)
-* [SDK for Go V2 Aurora reference](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/rds)
+- [Aurora User Guide](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/CHAP_AuroraOverview.html)
+- [Aurora API Reference](https://docs.aws.amazon.com/AmazonRDS/latest/APIReference/Welcome.html)
+- [SDK for Go V2 Aurora reference](https://pkg.go.dev/github.com/aws/aws-sdk-go-v2/service/aurora)
```

Comment:
This has to be rds, not aurora. I though these ref links were "sticky", maybe something changed? Either way, we need a way to override for specific cases like this where there's a "virtual" service.

Reply:
Done.